operator/src/main/java/oracle/kubernetes/operator/logging/LoggingFacade.java

```diff
@@ -651,6 +652,36 @@ public String getFormattedMessage(String msg, Object... args) {
     }
   }
 
+  /**
+   * Accessor for the resource bundle backing this logger.
+   * @return the bundle
+   */
+  public ResourceBundle getResourceBundle() {
+    for (Logger l = getUnderlyingLogger(); l != null; l = l.getParent()) {
+      ResourceBundle rb = l.getResourceBundle();
+      if (rb != null) {
+        return rb;
+      }
+    }
+    throw new AssertionError(formatMessage(MessageKeys.RESOURCE_BUNDLE_NOT_FOUND));
+  }
+
+  /**
+   * Formats message based on string loaded from the resource bundle backing this logger.
+   * @param msgId Message id
+   * @param params Parameters to message formatting
+   * @return Formatted message
+   */
+  public String formatMessage(String msgId, Object... params) {
```

Comment:
There is a getFormattedMessage() method in the same class that does the same thing, but with less error checking. We can get rid of that method and replace usages with this new method instead.

Reply:
Oh, thanks!  I missed it.  Let me check.

Reply:
I've decided to keep the formatMessage() method and remove the existing getFormattedMessage() method.